### PR TITLE
Upgrade to core v8

### PR DESCRIPTION
**Pull request by SzymonPobiega**

### CODE REVIEW

File: src/NServiceBus.Transport.SQS.AcceptanceTests/DelayedDelivery/SendOnly_when_configured_with_queue_delay_that_requires_multiple_cycles.cs [hunk 1]
```diff
@@ -13,7 +13,7 @@ public class SendOnly_when_configured_with_queue_delay_that_requires_multiple_cy
         public async Task Should_deliver()
         {
             var payload = "some payload";
-            var delay = QueueDelayTime.Add(QueueDelayTime).Add(TimeSpan.FromSeconds(1));
+            var delay = TimeSpan.FromSeconds(QueueDelayTime + QueueDelayTime + 1);
```
Comment (tmasternak) [inline src/NServiceBus.Transport.SQS.AcceptanceTests/DelayedDelivery/SendOnly_when_configured_with_queue_delay_that_requires_multiple_cycles.cs:16]:
```suggestion
            var delay = TimeSpan.FromSeconds(2*QueueDelayTime + 1);
```

**Comment by SzymonPobiega** [comment]
@tmasternak @timbussmann @tmasternak please approve (you might want to review, too :P)

**Comment by SzymonPobiega** [comment]
@danielmarbach would you mind take a look at my changes too? If you have some free time.

**Review by mauroservienti — APPROVED** [review]

File: src/NServiceBus.Transport.SQS/InputQueuePump.cs [hunk 2]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (danielmarbach) [inline src/NServiceBus.Transport.SQS/InputQueuePump.cs:1]: 🔥

File: src/NServiceBus.Transport.SQS/InputQueuePump.cs [hunk 3]
```diff
@@ -15,26 +15,33 @@ namespace NServiceBus.Transport.SQS
...
-
-            inputQueueUrl = await queueCache.GetQueueUrl(settings.InputQueue)
+            inputQueueUrl = await queueCache.GetQueueUrl(settings.ReceiveAddress)
                 .ConfigureAwait(false);
             errorQueueUrl = await queueCache.GetQueueUrl(settings.ErrorQueue)
                 .ConfigureAwait(false);
 
+            maxConcurrency = limitations.MaxConcurrency;
+
+
```
Comment (danielmarbach) [inline src/NServiceBus.Transport.SQS/InputQueuePump.cs:44]: 🔥

File: src/NServiceBus.Transport.SQS/InputQueuePump.cs [hunk 4]
```diff
@@ -391,29 +417,32 @@ async Task MovePoisonMessageToErrorQueue(Message message)
...
         int numberOfMessagesToFetch;
         ReceiveMessageRequest receiveMessagesRequest;
-        CriticalError criticalError;
-        string awsEndpointUrl;
+        CancellationTokenSource tokenSource;
 
-        static readonly TransportTransaction transportTransaction = new TransportTransaction();
-        static ILog Logger = LogManager.GetLogger(typeof(MessagePump));
+        static readonly TransportTransaction TransportTransaction = new TransportTransaction();
```
Comment (danielmarbach) [inline src/NServiceBus.Transport.SQS/InputQueuePump.cs:445]: Ups I think we have a bug here that is already present in the released version. This should not be static! Smells like almost a critical honestly

**Pull request merged**